routes/api/users.js

```diff
@@ -1,13 +1,95 @@
 import express from "express";
 import auth from "../../middleware/auth.js";
+import User from "../../models/schemas/userSchema.js";
 import { signUpUser } from "../../controllers/users/signUpUser.js";
 import { logInUser } from "../../controllers/users/logInUser.js";
 import { verifyUser } from "../../controllers/users/verifyUser.js";
 import { logOutUser } from "../../controllers/users/logOutUser.js";
 
+import fs from "fs/promises";
+import { fileURLToPath } from "url";
+import path, { dirname } from "path";
+import jimp from "jimp";
+import multer from "multer";
+
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = dirname(__filename);
+const AVATARS_FOLDER = path.join(__dirname, "../../public/avatars");
+const TMP_FOLDER = path.join(__dirname, "../../tmp");
+
 export const usersRouter = express.Router();
 
+const ensureFoldersExist = async () => {
+  try {
+    await fs.mkdir(AVATARS_FOLDER, { recursive: true });
+    await fs.mkdir(TMP_FOLDER, { recursive: true });
+  } catch (error) {
+    console.error("Error creating folders:", error);
+  }
+};
+
+ensureFoldersExist();
```

Comment:
W tej funkcji generalnie warto byłoby najpierw sprawdzić czy folder istnieje i jezeli nie to wtedy dopiero zabierać się za robotę :)